theme/static/js/hs_file_browser.js

```diff
@@ -22,36 +22,44 @@ function getFileTemplateInstance(fileName, fileType, logical_type, logical_file_
     var fileTypeExt = fileName.substr(fileName.lastIndexOf(".") + 1, fileName.length);
     var extIcon = "fa-file-o";
 
+    var iconTemplate =  "<span class='fb-file-icon fa fa-file-o'></span>";
```

Comment:
@mauriel133 A minor comment, but probably it is better to move ```iconTemplate =  "<span class='fb-file-icon fa fa-file-o'></span>";``` into the ```else``` phrase below for better readability and avoiding duplicate definition of iconTemplate variable?